docs/schedule/examples/fares-v2.md

```diff
@@ -216,7 +216,167 @@ The single ride fare product shown below has both `cash` and `tap-to-ride` fare
+
+In New York's MTA Metro-North railroad network, fares vary based on both the time of the day and the areas of the trips’s origin and destination. The following example illustrates the fare rules applicable to a trip from Grand Central Station to Poughkeepsie (NY, USA).
+
+This example based on a <a href="https://docs.google.com/spreadsheets/d/1-cD-R2OH5xAQAbNWNlrXD7WOw594lVdW-bomuLo6bI8/edit?usp=sharing" target="_blank">dataset</a> produced by <a href="https://www.itoworld.com/" target="_blank">ITO World</a>, features a trip that uses ten stops distributed in six different areas.
+
+[**stops.txt**](../../reference/#stopstxt)
+
+| stop_id | stop_name           | stop_lat  | stop_lon   |
+|---------|---------------------|-----------|------------|
+| ITO1669 | Peekskill           | 41.285103 | -73.930916 |
+| ITO1777 | Beacon              | 41.505814 | -73.984474 |
+| ITO1789 | New Hamburg         |  41.58691 | -73.947624 |
+| ITO1804 | Croton-Harmon       | 41.190002 | -73.882393 |
+| ITO1824 | Cortlandt           | 41.246258 | -73.921783 |
+| ITO1856 | Garrison            | 41.381126 | -73.947334 |
+| ITO1887 | Harlem-125th Street | 40.805256 | -73.939148 |
+| ITO1897 | Cold Spring         | 41.415382 | -73.958092 |
+| ITO2096 | Poughkeepsie        | 41.707058 |  -73.93792 |
+| ITO2383 | Grand Central       | 40.752823 | -73.977196 |
+
+
+[**stops_areas.txt**](../../reference/#stops_areastxt)
+
+| area_id   | stop_id |
+|-----------|---------|
+| mnr_1     | ITO1887 |
+| mnr_1     | ITO2383 |
+| mnr_HUD-5 | ITO1804 |
+| mnr_HUD-6 | ITO1669 |
+| mnr_HUD-6 | ITO1824 |
+| mnr_HUD-7 | ITO1856 |
+| mnr_HUD-7 | ITO1897 |
+| mnr_HUD-8 | ITO1777 |
+| mnr_HUD-8 | ITO1789 |
+| mnr_HUD-9 | ITO2096 |
+
+
+Service days for train services 3 and 13 are defined using calendar.txt. Notably, general service days (i.e. weekdays, weekends and anyday) are also defined, as these will be associated with specific timeframes in order to model `time-variable fares`.
+
+[**calendar.txt**](../../reference/#calendartxt)
+
+| service_id | monday | tuesday | wednesday | thursday | friday | saturday | sunday | start_date | end_date |
+|------------|--------|---------|-----------|----------|--------|----------|--------|------------|----------|
+| 13         | 1      | 1       | 1         | 1        | 1      | 0        | 0      | 20230612   | 20231006 |
+| 3          | 1      | 1       | 1         | 1        | 1      | 0        | 0      | 20230609   | 20231006 |
+| weekdays   | 1      | 1       | 1         | 1        | 1      | 0        | 0      | 20220101   | 20240101 |
+| weekends   | 0      | 0       | 0         | 0        | 0      | 1        | 1      | 20220101   | 20240101 |
+| anyday     | 1      | 1       | 1         | 1        | 1      | 1        | 1      | 20220101   | 20240101 |
+
+
+Specific time frames are created in `timeframes.txt`, including general service days (anytime, weekdays and weekends), and peak and off peak periods:
```

Comment:
Suggestion to avoid using "general service days" which could be confusing:

Records are created in `timeframes.txt`, including cases where the time covers the 24-hour range period (`anytime`, `weekdays` and `weekends`), and peak and off-peak periods: